networkd/network_server.ml

```diff
@@ -844,6 +844,35 @@ module Bridge = struct
 				Sysfs.bridge_to_interfaces name
 		) ()
 
+	let get_physical_interfaces _ dbg ~name =
+		Debug.with_thread_associated dbg (fun () ->
+			match !backend_kind with
+			| Openvswitch ->
+				let ifaces = List.filter (fun iface ->
+					String.startswith "eth" iface) (Ovs.bridge_to_interfaces name) in
+				if ifaces <> [] then ifaces
+				else
+					begin match (Ovs.bridge_to_vlan name) with
+					| Some (parent, vlan) ->
+						let ifaces = Ovs.bridge_to_interfaces parent in
+						List.filter (fun iface -> String.startswith "eth" iface) ifaces
+					| None -> []
+				end
+			| Bridge ->
+				let iface = List.hd (List.filter (fun iface ->
```

Comment:
The `List.hd` can fail (similar below). I think it would be better to use pattern matching here to access the first element and to deal with the situation when the list is empty.